### PR TITLE
Add workspace name parameter

### PR DESCRIPTION
**Pull request by roni-frantchi**
### Issue
Users should be able to set terraform workspace name.  

### Solution
Add CLI parameter

### CODE REVIEW

File: node/src/config/arguments.js [hunk 1]
```diff
@@ -49,6 +50,14 @@ const argumentsMap = {
     prompt: 'Project ID',
     group: ['deploy', 'destroy', 'approve', 'cancel']
   },
+  [WORKSPACE_NAME]: {
+    name: WORKSPACE_NAME,
+    alias: 'e',
```
Comment (avnerenv0) [inline node/src/config/arguments.js:55]: we already have `e` alias.
We don't have to have an alias for every option.

Comment (roni-frantchi) [reply node/src/config/arguments.js]: https://github.com/env0/env0-client-integrations/pull/53/commits/1294a05abe9dbf038de5a46d6a8b0cc7ef8aff6f

File: node/src/config/arguments.js [hunk 2]
```diff
@@ -49,6 +50,14 @@ const argumentsMap = {
     prompt: 'Project ID',
     group: ['deploy', 'destroy', 'approve', 'cancel']
   },
+  [WORKSPACE_NAME]: {
+    name: WORKSPACE_NAME,
+    alias: 'w',
+    type: String,
+    description: 'Terraform Workspace name - cannot be changed after the first deployment',
+    prompt: 'Terraform Workspace Name',
```
Comment (yaronya) [inline node/src/config/arguments.js:58]: `prompt` value is necessary only for options that are a part of the saved configuration

Comment (roni-frantchi) [reply node/src/config/arguments.js:58]: @yaronya so...?  
```suggestion
```

**Review by yaronya — APPROVED** [review]

**Pull request merged**